drivers/usb/device/usb_dc_nrfx.c

```diff
@@ -372,7 +372,7 @@ static inline struct usbd_ep_event *usbd_evt_alloc(void)
 			       sizeof(struct usbd_ep_event),
 			       K_NO_WAIT);
 	if (ret < 0) {
-		SYS_LOG_DBG("USBD event alloc failed!");
+		LOG_DBG("USBD event alloc failed!");
```

Comment:
We could use USB_ prefix, otherwise without function name logs are not very useful